src/m365/entra/commands/multitenant/multitenant-add.spec.ts

```diff
@@ -98,7 +101,8 @@ describe(commands.MULTITENANT_ADD, () => {
       throw 'Invalid request';
     });
 
-    await command.action(logger, { options: { displayName: 'Contoso organization', verbose: true } });
+    const parseResult = commandOptionsSchema.safeParse({ displayName: 'Contoso organization', verbose: true });
+    await command.action(logger, { options: parseResult.data });
```

Comment:
Let's use `safeParse` in validation tests and `parse` in normal tests. If the parsing fails, the test breaks.

```suggestion
    await command.action(logger, { options: commandOptionsSchema.parse({ displayName: 'Contoso organization', verbose: true }) });
```